examples/layers/widgets/gestures.dart

```diff
@@ -44,14 +42,14 @@ class _GesturePainter extends CustomPainter {
 
   @override
   bool shouldRepaint(_GesturePainter oldPainter) {
-    return oldPainter.zoom != zoom
-        || oldPainter.offset != offset
-        || oldPainter.swatch != swatch
-        || oldPainter.forward != forward
-        || oldPainter.scaleEnabled != scaleEnabled
-        || oldPainter.tapEnabled != tapEnabled
-        || oldPainter.doubleTapEnabled != doubleTapEnabled
-        || oldPainter.longPressEnabled != longPressEnabled;
+    return oldPainter.zoom != zoom ||
```

Comment:
IMHO the original version was easier to read.

Reply:
I tend to agree with you but the result is still quite readable.